README.md

```diff
@@ -1855,7 +1855,7 @@ The following extractors use this feature:
 #### twitter
 * `legacy_api`: Force usage of the legacy Twitter API instead of the GraphQL API for tweet extraction. Has no effect if login cookies are passed
 
-#### wrestleuniverse
+#### stacommu, wrestleuniverse
```

Comment:
@pukkandan is this how we want to handle two different extractors/sites having an identical extractor-arg?

Reply:
Looks fine to me. We can alway split it in future if needed.